react/src/App.tsx

```diff
@@ -5,85 +5,103 @@ import ModeSwitcher from "./components/ModeSwitcher";
-                dapibus. Maecenas sit amet rhoncus arcu. Sed sed molestie elit.
-                Nullam in interdum est, vitae aliquam ipsum. Nunc rutrum nibh ut
-                arcu egestas egestas.
+                consectetur interdum. Vestibulum condimentum, augue vitae lobortis
+                rhoncus, mi est ultricies mi, sed tincidunt magna nibh in lectus.
+                Pellentesque vel vulputate orci, vel lacinia orci. Sed suscipit
+                leo at diam ullamcorper, vitae volutpat neque dapibus. Maecenas
+                sit amet rhoncus arcu. Sed sed molestie elit. Nullam in interdum
+                est, vitae aliquam ipsum. Nunc rutrum nibh ut arcu egestas
+                egestas.
               </p>
+              <sdds-block>
+                <div className="sdds-headline-01">Headline</div>
+                <p className="sdds-body-01">
+                  Lorem ipsum dolor sit amet, consectetur adipiscing elit. In
+                  condimentum nisi ut eleifend ultrices. Nunc venenatis maximus
+                  sapien, ac bibendum nisl aliquam in. Morbi ac velit et ligula
+                  consectetur interdum. Vestibulum condimentum, augue vitae
+                  lobortis rhoncus, mi est ultricies mi, sed tincidunt magna nibh
+                  in lectus. Pellentesque vel vulputate orci, vel lacinia orci.
+                  Sed suscipit leo at diam ullamcorper, vitae volutpat neque
+                  dapibus. Maecenas sit amet rhoncus arcu. Sed sed molestie elit.
+                  Nullam in interdum est, vitae aliquam ipsum. Nunc rutrum nibh ut
+                  arcu egestas egestas.
+                </p>
+              </sdds-block>
             </sdds-block>
-          </sdds-block>
-        </div>
-        <div>
-        <div className="sdds-headline-02 sdds-u-pb1">Breadcrumbs</div>
-        <sdds-breadcrumb>
-          <sdds-breadcrumb-item href="#">Step one</sdds-breadcrumb-item>
-          <sdds-breadcrumb-item href="www.google.se">Step two</sdds-breadcrumb-item>
-          <sdds-breadcrumb-item current={true}>Current page</sdds-breadcrumb-item>
-        </sdds-breadcrumb>
+          </div>
+          <div>
+            <div className="sdds-headline-02 sdds-u-pb1">Breadcrumbs</div>
+            <sdds-breadcrumb>
+              <sdds-breadcrumb-item href="#">Step one</sdds-breadcrumb-item>
+              <sdds-breadcrumb-item href="www.google.se">Step two</sdds-breadcrumb-item>
+              <sdds-breadcrumb-item current={true}>Current page</sdds-breadcrumb-item>
+            </sdds-breadcrumb>
+          </div>
+          <div className='sdds-u-w-100'>
+            <div className="sdds-headline-02 sdds-u-pb1">Buttons</div>
+            <sdds-button
+                type="primary"
+                size="lg"
+                fullbleed
+                text="Primary fullbleed button"
+            ></sdds-button>
+            <div className="sdds-u-mt2 sdds-u-flex" style={{justifyContent: "space-around"}}>
```

Comment:
What do you think of introducing flex specific utility classes? 
For justify-content etc...

Reply:
Yep, could absolutely be something to look into. Would be nice to know how many are using the utility classes though?

Reply:
I agree with that as well ;) We would need to expand a bit on them and also raise awareness of their existence. I find them super useful when making layouts.